appeals/api/src/server/endpoints/integrations/integrations.validators.js

```diff
@@ -0,0 +1,36 @@
+import { readdirSync, readFileSync } from 'node:fs';
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
+
+const SCHEMA_PATH = './src/message-schemas';
+const loadSchemas = async () => {
+	const schemas = await readdirSync(SCHEMA_PATH)
+		.filter((file) => file.endsWith('.schema.json'))
+		.map((file) => {
+			return JSON.parse(readFileSync(`./src/message-schemas/${file}`).toString());
+		});
+
+	return schemas;
+};
+export const schemas = {
+	appellantCase: 'appellant-case-submission',
+	lpaQuestionnaire: 'questionnaire-submission',
+	document: 'document'
+};
+
+export const validateFromSchema = async (
+	/** @type {string} */ schema,
+	/** @type {any} */ payload
+) => {
+	const schemas = await loadSchemas();
+	const ajv = new Ajv({ schemas });
+	addFormats(ajv);
+
+	const validator = ajv.getSchema(`${schema}.schema.json`) || (() => false);
+	if (!validator(payload)) {
+		// @ts-ignore
```

Comment:
We'd probably avoid this if the function had some jsdoc to define the types it needs/returns.

Reply:
yep, just needed to make sure we check that the validator function is not undefined... Solved.

Reply:
Sorry I meant avoid the `@ts-ignore`, if we defined the return type.

Reply:
yep, that's been removed as the check ensures the type...